### PR TITLE
Changes for number of log messages and badge color

### PR DESCRIPTION
**Pull request by adidas**
This PR has integrated the comments made in issue #94. The only spot that I think might be questionable is my use of the progress-bar-XXX css class.

When no errors or warnings are present but info logs are the logging panel tab now looks like
20170712_pdbt_info

When a warning is present it looks like
20170712_pdbt_warn

When an error is present it looks like
20170712_pdbt_error

Note that the error count will not include the counts of any lesser levels. However, the error level is sum of criticals and errors.

### CODE REVIEW

**Comment by bertjwregeer** [comment]
Could you please merge master into your branch (or rebase, as you wish) and re-push your changes?

Thank you!

**Comment by adidas** [comment]
So, I think I did it...let me know if not.

**Comment by bertjwregeer** [comment]
Yep, looks good now!

File: src/pyramid_debugtoolbar/panels/__init__.py [hunk 1]
```diff
@@ -68,6 +68,8 @@ class DebugPanel(object):
     #: when the user clicks on the panel's tab in the toolbar.
     url = ''
 
+    nav_subtitle_bg_color = ''
```
Comment (mmerickel) [inline src/pyramid_debugtoolbar/panels/__init__.py]: Add a documentation comment here using the `#: ` syntax.

**Comment by mmerickel** [comment]
My last comment is that we should define our own css classes to be used but it's ok if we don't do that right now.

**Comment by adidas** [comment]
Made some changes. Let me know what you think. Thanks for the feedback!

**Comment by mmerickel** [comment]
LGTM. I'll probably rename `nav_subtitle_bg_color` to `nav_subtitle_style` or something prior to releasing this but that's super minor.

**Pull request merged**